### PR TITLE
Meshford patch 2

### PR DESCRIPTION
**Pull request by Meshford**

### CODE REVIEW

File: examples/torch/object_detection/README.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ This sample demonstrates DL model compression capabailites for object detection
 
 ## Installation
 
+At this point it is assumed that you have already installed nncf
```
Comment (vshampor) [inline examples/torch/object_detection/README.md:14]: Add colon at the end

File: examples/torch/object_detection/README.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ This sample demonstrates DL model compression capabailites for object detection
...
+
+```
+pip install nncf
```
Comment (vshampor) [inline examples/torch/object_detection/README.md:17]: You are only referencing the PyPI path. Should also reference all the other ways to install NNCF if you are mentioning this; also, should update all other READMEs for the examples.

**Review by vshampor — CHANGES_REQUESTED** [review]

**Comment by FyodorK** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

File: examples/torch/object_detection/README.md (hunk 2 above)
Comment (Meshford) [reply examples/torch/object_detection/README.md]: done

File: examples/torch/object_detection/README.md (hunk 1 above)
Comment (Meshford) [reply examples/torch/object_detection/README.md]: done

**Review by vshampor — APPROVED** [review]

**Pull request merged**